src/app.js

```diff
@@ -0,0 +1,73 @@
+const displayTasks = document.querySelector('#task_list');
+const form = document.getElementById('form');
+const newTask = document.querySelector('#add_task');
+
+let taskList = [];
+
+function showTasks() {
+  displayTasks.innerHTML = '';
+  taskList = JSON.parse(localStorage.getItem('storedTask')) || [];
+  taskList.forEach((item) => {
+    const comp = item.completed ? 'checked' : '';
+    const task = document.createElement('div');
+    displayTasks.appendChild(task);
+    task.classList.add('task');
+    task.innerHTML = `
+    <input type="checkbox" class="check" ${comp} id="check" data-set="${item.index}">
```

Comment:
Nice checkbox ☑️ but it's not updating the `localStorage` when it's clicked, when the checkbox is clicked, the `localStorage` should update the `completed` attribute of the current object (index) 🙃 so that when the page is refreshed you still get the same value that you had before, for you now it's always false when the page is refreshed for all the elements. ❌